tests/copy-paste.js

```diff
@@ -685,13 +685,13 @@ define([
             paste([
                 ["id", "type", "labelItext:en-default", "itemsetData"],
                 ["/select", "SelectDynamic", "select",
-                 '[{"instance":null,"nodeset":"/items","labelRef":"@name","valueRef":"@id"}]'],
+                 '[{"instance":null,"nodeset":"/items","labelRef":"name","valueRef":"@id"}]'],
```

Comment:
this should probably not change, I suspect it's revealing a bug if this test is now failing. It may be a bug in test setup: may need to ensure data sources are loaded prior to running this test. My guess is that it's related to [`afterDynamicSelectInsert()`](https://github.com/dimagi/Vellum/blob/9a7b64b1e16d100b18c094fcc944471a13cfe7bb/src/itemset.js#L166) calling [`getDatasources()`](https://github.com/dimagi/Vellum/blob/9a7b64b1e16d100b18c094fcc944471a13cfe7bb/src/itemset.js#L359-L360), which returns asynchronously loaded data. It seems possible that the default value if the data sources are not yet loaded is `"name"`, and that's why `"@name"` is not pasted/copied correctly in this test (any value other than `"name"` will likely fail).